pkg/provider-local/charts/shoot-system-components/charts/local-path-provisioner/templates/deployment.yaml

```diff
@@ -18,7 +18,8 @@ spec:
     metadata:
       labels:
         app: local-path-provisioner
-        networking.gardener.cloud/to-runtime-apiserver: allowed
```

Comment:
Hi @rfranzke 👋 

We noticed that this label was added in https://github.com/gardener/gardener/pull/11892/commits/18a7532a29641c4845595d7b7f0cb526923ea6b3#diff-497904ac0d5083098fb661cb58a230fc3d9cf75c6a2aef4c740e00cd72268f04R21.
We were not sure if it is needed. Maybe it is needed for autonomous Shoot cluster scenario but we had issues with the local setup (reported them in https://github.com/gardener/gardener/issues/12210) and could not confirm/reject the assumption.

Could you confirm if we can remove this label? Thanks in advance!